src/commands/farm.rs

```diff
@@ -26,6 +26,14 @@ pub(crate) async fn farm(is_verbose: bool) -> Result<()> {
     install_tracing(is_verbose);
     color_eyre::install()?;
 
+    let instance = SingleInstance::new("subspaceFarmer")
```

Comment:
Why should we have a single instance of the farmer? We supposed to be able to start 2 cli applications if they are on different chains.

Reply:
this is a great suggestion. I've added a TODO in the code for that. It will be a nice to have.